conseil-api/src/main/resources/application.conf

```diff
@@ -34,20 +34,19 @@ conseil {
 }
 
 akka {
-
-  # this is essentially available to enable composition of database operations
-  # current configuration is based upon the default-blocking-io-dispatcher in akka
-  tezos-dispatcher {
-    type: "Dispatcher"
-    executor: "thread-pool-executor"
-    throughput: 1
-
-    thread-pool-executor {
-      fixed-pool-size: 16
+  http {
+    # this is essentially available to enable composition of database operations
+    # current configuration is based upon the default-blocking-io-dispatcher in akka
+    dispatcher {
```

Comment:
will this conflict and override a default akka-http value that might be used elsewhere from the toolkit?

Reply:
In akka-http configuration, section like `dispatcher` does not exist, but I can rename that to feel a bit safer or move under different section. Which would you prefer? I guess renaming that simply to `api-dispatcher` or `data-api-dispatcher` would do the trick.

Reply:
no need for me, I was just double-checking with you